Automattic/newspack-network
# feat(event-log): collapse data

The content distribution project introduces events with significantly larger payloads shared between network sites. Because of that, we need better handling of our event log table.

This PR proposes collapsing the data contents inside a disabled textarea with a "Copy to clipboard" button. The idea is to suggest parsing that information somewhere else, which will always be more suitable than any UI we could produce here.

<img width="1333" alt="image" src="https://github.com/user-attachments/assets/2d8f7432-4982-4924-abbd-eed9c8935645" />

### Testing

1. Visit the Event Log page in the hub
2. Clip "Copy to clipboard" on any item
3. Confirm it pastes the contents
